### DISCUSSION TITLE
v2.8.0

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#990] [Versions]
This version comes with added support for the between batch protocol and a phases widget added to the Comparator.

---
Release Sponsor: [Coffee Machines Sale](https://cmsale.com/)  
<a target="_blank" href="https://cmsale.com/">
<img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/cms-logo.jpg?raw=true" width="500"></a>
---

The [Artisan project](https://artisan-scope.org/) runs on donations from individuals and companies recognising the value of Artisan for their work or leisure. Consider supporting this project with your [donation](https://www.paypal.com/paypalme/MarkoLuther), or even better, purchase an [artisan.plus membership](https://shop.artisan.plus/)!

## Documentation

__>>>  [Release Notes](https://artisan-roasterscope.blogspot.com/2022/10/artisan-v280.html) <<<__

See the [detailed release history](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/ReleaseHistory.md) for a complete list of additions, changes and fixes and the [Installation Instructions](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/Installation.md) for platform specific notes. Read our [Quick-Start Guide](https://artisan-scope.org/docs/quick-start-guide/)!

## Supported Platforms

Builds are available for macOS Catalina 10.15, Windows 10 (x64), Linux glibc 2.31 and 32bit Raspbian Bullseye or newer.

Legacy builds for macOS High Sierra 10.13, Windows 8 (x64) and Raspbian Bullet are available too.

---

PLEASE FOLLOW AND TAG US!  
&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.facebook.com/ArtisanScope"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/facebook-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.instagram.com/artisanscope/"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/instagram-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.youtube.com/channel/UCEp-DH_Y7sBR3-X3FWomgog"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/youtube-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://twitter.com/artisanscope"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/twitter-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.linkedin.com/company/artisan-plus"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/linkedin-svgrepo-com.svg?raw=true" width="30"></a>
---

__Licence Notes__

_Artisan is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version._

_Artisan is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE._

_Copies of the GNU General Public License has been included with this 
distribution in the file `LICENSE.txt`. An online version is available at
<http://www.gnu.org/licenses/>._

<hr /><em>This discussion was created from the release <a href='https://github.com/artisan-roaster-scope/artisan/releases/tag/v2.8.0'>v2.8.0</a>.</em>

**Comment by marufwahyup** [comment]
I want to inform you that I got an error while using Modbus. I'm using Arduino mega as a slave with the atmega16u2 serial, but if I'm using Arduino mega with ch340g serial the Modbus will work flawlessly. Is there any solution to it? Thanks in advance

**Comment by MAKOMO** [comment]
If you discover an issue in the software please report this under Issues. You don't give any details, not even which Artisan version you are running, which makes it hard to help. If I understand correctly things work with one USB2serial connector but not with the other. Might be a hardware issue rather than a software one, right?

**Reply by marufwahyup** [reply]
But if I'm using the old one like 1.1.0 they're all fine. I've already made an issue report would you mind to answer it there?
